File: app/src/main/java/com/flask/colorpicker/sample/PrefsActivity.java
package com.flask.colorpicker.sample;

import android.os.Bundle;
import android.preference.PreferenceActivity;

public class PrefsActivity extends PreferenceActivity {
	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);

		addPreferencesFromResource(R.xml.prefs);
	}
}

File: app/src/main/java/com/flask/colorpicker/sample/PrefsActivity.kt
package com.flask.colorpicker.sample

import android.os.Bundle

import androidx.preference.PreferenceFragmentCompat

class PrefsActivity : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.prefs, rootKey)
    }
}